common/web/lm-worker/src/correction/distance-modeler.ts

```diff
@@ -582,17 +644,22 @@ namespace correction {
 
       let batcher = new BatchingAssistant();
 
+      const timer = new ExecutionTimer(maxTime*3, maxTime);
```

Comment:
Why `maxTime*3` vs `maxTime`? Is that likely to be a constant multiplier? If so, we should build that into the class I should think.

Reply:
... because I needed to set the max true-system-time threshold to something larger than the max execution time, and _something_ based on the max execution time seemed to make the most sense.  I just picked '3' for the coefficient.

Could probably be `maxTime*2` pretty safely, and I could entertain arguments for other values.  Maybe even 1.5; we don't expect heavy "thrashing" behavior from constant OS context switches.

The key interpretation:  we're willing to wait up to 0.1 sec for predictions if execution gets delayed due to context switches, but for the common case we should only need to wait 0.033 sec (when no context switches occur).  That 0.033 sec value has long been set via constant.

Reply:
OK. The numbers are fairly off-the-cuff, but we need to start somewhere. Measuring and testing on real low-end Android devices may be helpful.